linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/CoreJSLib.scala

```diff
@@ -1430,12 +1509,14 @@ private[emitter] object CoreJSLib {
       val initSpecialArray = {
         val componentData = varRef("componentData")
         val arrayClass = varRef("arrayClass")
+        val typedArrayClass = varRef("typedArrayClass")
```

Comment:
Would it make sense to set the `typedArrayClass` as a static field on the `arrayClass`? (TBH I'm not sure what that would imply, it's just an idea).

Reply:
I don't think that would simplify anything.